eng/common/scripts/common.ps1

```diff
@@ -20,12 +20,43 @@ $EngScriptsLanguageSettings = Join-path $EngScriptsDir "Language-Settings.ps1"
 if (Test-Path $EngScriptsLanguageSettings) {
   . $EngScriptsLanguageSettings
 }
-If ($LanguageShort -eq $null)
+if ($null -eq $LanguageShort)
 {
   $LangaugeShort = $Language
 }
+if ($null -eq $isDevOpsRun)
+{
+  $isDevOpsRun = ($null -ne $env:SYSTEM_TEAMPROJECTID)
+}
 
 # Transformed Functions
 $GetPackageInfoFromRepoFn = "Get-${Language}-PackageInfoFromRepo"
 $GetPackageInfoFromPackageFileFn = "Get-${Language}-PackageInfoFromPackageFile"
-$PublishGithubIODocsFn = "Publish-${Language}-GithubIODocs"
\ No newline at end of file
+$PublishGithubIODocsFn = "Publish-${Language}-GithubIODocs"
+
+function LogHelper ($logType, $logArgs)
+{
+  if ($isDevOpsRun) 
+  {
+    Write-Host "##vso[task.LogIssue type=$logType;]$logArgs"
+  }
+  else 
+  {
+    Write-Warning "$logArgs"
```

Comment:
We don't want everything to fallback to Write-Warning.

Reply:
We should use the natural fallback which is Write-Error, Write-Warning, Write-Debug